pkg/plugin/aggregation/aggregator_test.go

```diff
@@ -94,7 +94,7 @@ func TestAggregation_noExtension(t *testing.T) {
 
 func TestAggregation_tarfile(t *testing.T) {
 	expected := []plugin.ExpectedResult{
-		{ResultType: "e2e"},
+		{ResultType: "e2e", NodeName: "global"},
```

Comment:
I've done the "nodename=global" work but it does have the tiny problem of: what if someone actually has a node with that name? Does it make a problem?

So it seems like it might but we actually err out if two plugins have the same `ResultType` so there isn't a risk of path conflics.

The only concern is when parsing the results will we interpret `e2e/results/global` as a job with results or a daemonset with results from a single node named `global`.

Since those are really just problems with reading folder names it doesn't seem like we can get away from _some_ ambiguity so it seems fine to me. I just wanted to call it out so another brain can at least consider it.

Reply:
We are dumping all the plugin info into a config map with a known name; we can also read that in and iterate over the data. You're correct that is the most accurate thing to automate; on the next PR I may do that as well but I do think that it helps to have the path also provide implicit context so that you don't have to fully parse a separate file at a known location to get that information.